# EuropeWildboarGenome

The de novo assembly of a European wild boar genome revealed unique patterns of chromosomal structural variations and segmental duplications

Jianhai Chen, Jie Zhong, Xuefei He, Xiaoyu Li, Pan Ni, Toni Safner, Nikica Šprem, Jianlin Han


First published: 02 March 2022 https://doi.org/10.1111/age.13181

Abstract

The rapid progress of sequencing technology has greatly facilitated the de novo genome assembly of pig breeds. However, the assembly of the wild boar genome is still lacking, hampering our understanding of chromosomal and genomic evolution during domestication from wild boars into domestic pigs. Here, we sequenced and de novo assembled a European wild boar genome (ASM2165605v1) using the long-range information provided by 10× Linked-Reads sequencing. We achieved a high-quality assembly with contig N50 of 26.09 Mb. Additionally, 1.64% of the contigs (222) with lengths from 107.65 kb to 75.36 Mb covered 90.3% of the total genome size of ASM2165605v1 (~2.5 Gb). Mapping analysis revealed that the contigs can fill 24.73% (93/376) of the gaps present in the orthologous regions of the updated pig reference genome (Sscrofa11.1). We further improved the contigs into chromosome level with a reference-assistant scaffolding method. Using the ‘assembly-to-assembly’ approach, we identified intra-chromosomal large structural variations (SVs, length >1 kb) between ASM2165605v1 and Sscrofa11.1 assemblies. Interestingly, we found that the number of SV events on the X chromosome deviated significantly from the linear models fitting autosomes (R2 > 0.64, p < 0.001). Specifically, deletions and insertions were deficient on the X chromosome by 66.14 and 58.41% respectively, whereas duplications and inversions were excessive on the X chromosome by 71.96 and 107.61% respectively. We further used the large segmental duplications (SDs, >1 kb) events as a proxy to understand the large-scale inter-chromosomal evolution, by resolving parental-derived relationships for SD pairs. We revealed a significant excess of SD movements from the X chromosome to autosomes (p < 0.001), consistent with the expectation of meiotic sex chromosome inactivation. Enrichment analyses indicated that the genes within derived SD copies on autosomes were significantly related to biological processes involving nervous system, lipid biosynthesis and sperm motility (p < 0.01). Together, our analyses of the de novo assembly of ASM2165605v1 provides insight into the SVs between European wild boar and domestic pig, in addition to the ongoing process of meiotic sex chromosome inactivation in driving inter-chromosomal interaction between the sex chromosome and autosomes.

https://onlinelibrary.wiley.com/doi/10.1111/age.13181

DOI: 10.1111/age.13181

The genome annotation files could be downloaded here:

https://drive.google.com/drive/folders/1JBD6FMwKtImZ6TlxfHaMVPcFc0f1Ppk6?usp=drive_link

03.ncRNA/:
Sus_scrofa.rRNA_cmscan.gff

Sus_scrofa.snRNA.gff

Sus_scrofa.miRNA.gff

Sus_scrofa.tRNA.gff

Sus_scrofa.60tRNAdb

Sus_scrofa.ncRNA.statistics.xls

Sus_scrofa.rRNA.gff


04.function/:


01.repeat/:
final.library
Sus_scrofa.RepeatProteinMask.gff

Sus_scrofa.TRF.gff

Sus_scrofa.known.RepeatMasker.gff

Sus_scrofa.known.RepeatMasker.out

Sus_scrofa.denovo.RepeatMasker.gff

Sus_scrofa.denovo.RepeatMasker.out

Sus_scrofa.repeat.statistics.xls

Sus_scrofa.repeat.known.sequence.divergence.svg

Sus_scrofa.repeat.known.sequence.divergence.png

Sus_scrofa.repeat.denovo.sequence.divergence.svg

Sus_scrofa.repeat.denovo.sequence.divergence.png

all_without_trf.gff


Gene Annotation

Ssc_EuropeanWildboar.coding.gene.V1.20210311.gff, https://drive.google.com/file/d/1vcwyrYihpZGSmN40lEsQgj_k8mo31m8C/view?usp=sharing

Ssc_EuropeanWildboar.coding.gene.V1.20210311.cds, https://drive.google.com/file/d/1B8Mr8f05XNvMZZ7GXPc1aO8846MNgFrS/view?usp=drive_link

Ssc_EuropeanWildboar.coding.gene.V1.20210311.pep, https://drive.google.com/file/d/1GmyBEEmRbuKB5fWZgpYrpiQrunesmiW0/view?usp=drive_link

jianhaichen@uchicago.edu

